project_resource_calendar/tests/test_instrument.py

```diff
@@ -0,0 +1,37 @@
+# © 2018 Savoir-faire Linux
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/LGPL).
+
+from odoo.addons.project_resource_calendar.tests.common import TestCalendarEventCommon
+
+
+class TestInstrument(TestCalendarEventCommon):
+
+    def setUp(self):
+        super(TestInstrument, self).setUp()
+        self.check_record_count = 2
+        self.instrument_name = 'Instrument_name_for_auditlog_testing'
+        self.resource_type_name = 'material'
+        self.double_book_status = True
+        self.model_name = 'project_resource_calendar.' \
+                          'model_resource_calendar_instrument'
+
+        self.room_info = self.Rooms.create({
+            'name': 'Test Room 1',
+            'resource_type': 'room',
+            'allow_double_book': True,
```

Comment:
Why some time you use the self.double_book_status and some time you directly put True. I prefer that you directly put True or False without using a variable.

Reply:
@rimbendhaou :Done